en-US/Players_Guide/03_Creating_a_Character/Cultures/Cultures/Tunnel_Halfling.md

```diff
@@ -1,4 +1,21 @@
 #### Tunnel Halfling
 
-> **Warning:**
-> This culture has not yet been transferred from the Google Doc.
+This culture thrives in tight spaces and under pressure.
+Its members value freedom, cunning, and quick thinking to overcome obstacles.
+\
+\
+**Fleet of Foot.**
+Your speed increases by 5 feet.
+\
+**Rebellious Tactics.**
+You gain proficiency with one of the following skills: Acrobatics, Deception, Nature, or Stealth.
+\
+**Slippery.**
+When you are grappled, you can use your reaction to automatically escape that grapple.
+Once you have used this trait, you cannot do so again until you finish a short or long rest.
```

Comment:
This sounds like a "once per short or long rest" ability to me. Is that correct?